### DISCUSSION TITLE
Welcome to CBRG Discussions!

### DISCUSSION THREAD

**Discussion by BAREJAA** [BAREJAA/CBRG#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SophBr** [comment]
Hi everyone!
I'm Sophie and I study "Genes in Behavior and Health" in Amsterdam!
It is a Psychology program that focuses on the genetic basis of psychological phenotypes.
I'm really excited to meet some like-minded people here and learn about what everyone else is working on!

**Reply by BAREJAA** [reply]
Hi Sophie! Welcome. In case you missed it, here's a link to tomorrow's livestream (starting at 11 AM ET) - https://www.youtube.com/watch?v=O9J9DZFEGQQ
